lib/claude/process-manager.ts

```
@@ -0,0 +1,450 @@
+  async createSession(config: ProcessConfig): Promise<SessionMapping> {
+    if (this.processes.size >= this.MAX_CONCURRENT_SESSIONS) {
+      throw new Error('Maximum concurrent sessions reached');
+    }
+
+    const webSessionId = uuidv4();
+    const args = this.buildCliArgs(config);
+    const env = this.buildEnvironment(config.authConfig);
+
+    try {
+      const process = spawn('claude', args, {
+        env: { ...process.env, ...env },
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)